modules/gapi/src/backends/cpu/gnnparsers.cpp

```diff
@@ -246,6 +246,33 @@ void parseSSD(const cv::Mat&  in_ssd_result,
     }
 }
 
+static void checkYoloDims(const MatSize& d) {
+    // Accept 1x1x1xN, 1x1xN, 1xN,
+    // 1x13x13xN and 13x13xN
+    GAPI_Assert(d.dims() >= 1);
+    const auto checkOneDimCase = [&](){
+        GAPI_Assert(d[d.dims()-1]%(5*13*13) == 0);
+        for (int i = 0; i < d.dims()-2; i++) {
```

Comment:
``d.dims() - 1`` should be as a limit

Reply:
Please check 1x1x3x21125, I think it should be acceptable with the current algorithm

Reply:
That's a nice catch, fixed, thanks!